fe/fe-core/src/main/java/com/starrocks/lake/delete/LakeDeleteJob.java

```diff
@@ -102,9 +103,15 @@ public void run(DeleteStmt stmt, Database db, Table table, List<Partition> parti
                     beToTablets.size());
             SystemInfoService systemInfoService = GlobalStateMgr.getCurrentSystemInfo();
             for (Map.Entry<Long, List<Long>> entry : beToTablets.entrySet()) {
-                Backend backend = systemInfoService.getBackend(entry.getKey());
+                // TODO: need to refactor after be split into cn + dn
+                ComputeNode backend = systemInfoService.getBackend(entry.getKey());
```

Comment:
can we add new interface to SystemInfoService, called `getComputeNodeExtended()` which search backend first and then if shared_data mode, search from compute node as well. 

this way when we done the be split. we only change this interface, not all the places around.